### ISSUE TITLE
URL Params on separate line break if HTTP version specified

### ISSUE THREAD

**Issue by nstansby** [Huachao/vscode-restclient#621]
- REST Client Version: 0.24.1
- VSCode Version: 1.46.1
- OS Version: Win 10 Pro, v 1909, build 18363.900

Steps to Reproduce:

1. Send a request with a URL parameter specified inline, such as this:
```
GET {{hubspotApiUrl}}/crm/v3/objects/contacts/41753?hapikey={{hubspotApiKey}} HTTP/1.1
```
The server *will* receive the URL parameter.

2. Change the request to put the URL parameters on separate lines, like this:
```
GET {{hubspotApiUrl}}/crm/v3/objects/contacts/41753 HTTP/1.1
?hapikey={{hubspotApiKey}}
```
The server *will not* get the `hapikey` parameter.

3. Remove the HTTP version specified, like this:
```
GET {{hubspotApiUrl}}/crm/v3/objects/contacts/41753
?hapikey={{hubspotApiKey}}
```
The server *will* get the `hapikey` parameter.

The format used in example 2 used to work (not sure when it broke, but recently) and I have a LOT of .rest files that follow this format!

I looked at [Issue 607](https://github.com/Huachao/vscode-restclient/issues/607) which is similar, but this seems to be slightly different (and still broken).

**Comment by Huachao** [comment]
@nstansby sorry for the **breaking change**, that if you use the multi-line query string, the HTTP version information should be put at the end of the URL, that means right after the last query string line. https://github.com/Huachao/vscode-restclient/issues/607#issuecomment-645395008

**Comment by nstansby** [comment]
Thanks for the quick response @Huachao!  Sorry, I missed that comment even though I looked at the issue.  I understand the issue with supporting blank lines in the request body, and why it required a breaking change with a blank line in a request,  but I don't understand why it broke this functionality.  I assume since you closed the issue, it's going to stay this way?  It's really unfortunate, because I often edit the URL parameters, while the URL and HTTP version pretty much never change.

**Comment by Huachao** [comment]
@nstansby The reason for doing this change(*Put HTTP version after the last query params*) is not related to the blank lines in request body issue. The actual reason is that I did some refactoring that I will join all the request lines(including query lines) directly and pass the joined URL. So here if you put the HTTP version at the end of the request line, the joined result will lead to an invalid URL. I know that we can do some testing, that if user put the HTTP version after the request URL line, the HTTP version will be adjusted to the end of the joined result by my code. I just think the code doesn't smell well 😆 .

**Comment by nstansby** [comment]
Understood.  To be honest when I started using REST Client, I found the old syntax a bit strange (because the things I specified got put together, but in a different order!) but I quickly got used to it, and liked it... again, because I never change the URL or HTTP version, but I tweak URL params all the time.

I've just removed the HTTP/1.1 from all my .rest files; I was really just putting it there for completeness.
